tensorflow-mnist-input-data/Setup.hs

```diff
@@ -33,6 +33,9 @@ import System.FilePath ((</>))
 import System.Directory (doesFileExist)
 import qualified Crypto.Hash as Hash
 import qualified Data.ByteString.Lazy as B
+import qualified Data.ByteString.Char8 as C
```

Comment:
I'm somewhat wary of treating binary data as Char8. Any chance this can be kept as Data.ByteString.Lazy? The Browser API is very general and I can't immediately tell if it's possible.